app/models/grda_warehouse/enrollment_change_history.rb

```diff
@@ -6,17 +6,30 @@
 
 module GrdaWarehouse
   class EnrollmentChangeHistory < GrdaWarehouseBase
-    belongs_to :client
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client'
```

Comment:
this relation threw an error, rails couldn't resolve "client" in the current namespace